src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -483,4 +485,59 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
+    /**
+     * Removes templates and actual values filled in place of templates
+     * for status messages
+     * 
+     * Example : "You are enrolled in %s" and "You are enrolled in CS2103"
+     *            are converted to "You are enrolled in "
+     *            
+     * @param statusMessage
+     * @return status message without variables and actual values
+     */
+    public static String removeVariables(String statusMessage) {
```

Comment:
It's a bit hackish having to do this. You wont need it if you introduce a new `StatusMessage` class. The importance of the message can be decided at the point you instantiate the object, not afterwards.